src/wfs/parseParams.js

```diff
@@ -81,9 +81,21 @@ export function parseWfsParams (req, res) {
     }
   }
 
+  let count = null
+  if (req.query.count != null) {
+    count = parseInt(req.query.count[0])
```

Comment:
Look into this, why is it referenced as an array.

Reply:
Referenced as an array to prevent syntax error. `req.query.count` is of type `ParsedQs` which cannot be passed into `parseInt()`. For clarity switched to using `toString()`.